src/mrinufft/extras/gradient.py

```diff
@@ -47,16 +47,23 @@ def cg(operator, kspace_data, x_init=None, num_iter=10, tol=1e-4):
     velocity = tol * velocity + grad / Lipschitz_cst
```

Comment:
```suggestion
    velocity = tol * velocity + grad / lipschitz_cst
```

Please make the same changes bellow..

Reply:
The variable should start with short letter.